pkg/addons/addons.go

```diff
@@ -199,7 +201,13 @@ https://github.com/kubernetes/minikube/issues/7332`, out.V{"driver_name": cc.Dri
 				return errors.Wrap(err, "registry port")
 			}
 			if enable {
-				out.Styled(style.Tip, `Registry addon on with {{.driver}} uses {{.port}} please use that instead of default 5000`, out.V{"driver": cc.Driver, "port": port})
+				str := out.Sprintf(style.None, `Registry addon with {{.driver}} driver uses port {{.port}} please use that instead of default port 5000`, out.V{"driver": cc.Driver, "port": port})
```

Comment:
what do you think about a helper that makes the code look better ?

for example
```
out.BoxNotice(Style, "message to be printed in the box") 
```
so we dont have to repeat this for all other thigns ?

we could use this BoxNotice for important notices that most users usually ignore and they keep asking in the issues (even though we had sent suggestion)

Reply:
Sure. I'm thinking that zero padding came out a bit wrong, anyway